src/components/content/resource/history.ts

```diff
@@ -150,3 +147,149 @@ function wasEnteredInError(resource: fhir4.FhirResource) {
 
   return false;
 }
+
+function getResourceFQSQuery(resourceType: ResourceTypeString) {
+  switch (resourceType) {
+    case "Condition":
+      return conditionsQuery;
+    case "AllergyIntolerance":
+      return allergyQuery;
+    default:
+      return "";
```

Comment:
Let's make sure we don't silently error if we accidentally use this when it isn't setup.
```suggestion
      throw new Error(`Resource type to FQS query not implemented yet for ${resourceType}`);
```